package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@solana/spl-token-registry",
-  "version": "0.2.81",
```

Comment:
you dont need to update version that happens automatically.

Reply:
Not sure how this got in here - the commit `fe93904` was added after I opened the PR 🤔

Reply:
Ah I see - the GH workflow ran automatically in my forked repo. Disabled them.